src/analog.jl

```diff
@@ -11,22 +8,22 @@ analog_input_configs = Dict{AbstractString,Number}(
 
 create an analog input channel, and a new task if one is not specified
 """
-function analog_input(channel::String; terminal_config="differential", range=nothing)
+function analog_input(channel::String; terminal_config::TerminalConfig=NRSE, range=nothing)
```

Comment:
changing the default terminal configuration would be breaking, so we'd have to bump the minor version number.  is there a reason you advocate for this change?  what is the default when using LabView?

Reply:
I guess I'm not sure. Differential as default is probably less troublesome. Will roll that in. As an aside though, isn't changing the signature already breaking?